ofl/orbit/METADATA.pb

```diff
@@ -0,0 +1,22 @@
+name: "Orbit"
+designer: "Sooun Cho"
+license: "OFL"
+category: "SANS_SERIF"
```

Comment:
The problem is that this font is not really monospace. The Latin glyphs all have a width of 600, and the Korean all have a width of 900. So it isn't monospace all up, nor does it conform to the 2:1 doublewidth standard for monospace fonts. 

For most users, this would be considered monospace, but I think it more accurate to say it isn't.